packages/TorneloScoresheet/src/pages/EnterPgn/EnterPgn.tsx

```diff
@@ -76,45 +46,80 @@ const EnterPgn: React.FC = () => {
-              </PrimaryText>
-              <View style={styles.inputBoxesContainer}>
-                {inputBoxesContent.map((inputBoxContent, key) => (
-                  <InputBox
-                    key={key}
-                    style={styles.inputBox}
-                    value={inputBoxContent[0]}
-                    onChangeText={inputBoxContent[1]}
-                    placeholder={inputBoxContent[2]}
-                  />
-                ))}
-              </View>
-            </View>
+        <View style={styles.container}>
+          <View style={styles.informationAndInputBoxContainer}>
+            <PieceAsset
+              piece={{ type: PieceType.King, player: PlayerColour.Black }}
+              size={150}
+              colour={colours.secondary}
+              style={styles.piece}
+            />
+            <PrimaryText
+              style={styles.title}
+              size={38}
+              weight={FontWeight.SemiBold}
+              label="Arbiter Mode"
+            />
+            <EditCurrentEvent
+              display={showEditEvent}
+              dismiss={() => setShowEditEvent(false)}
+            />
+            {arbiterInfo !== null && (
+              <RoundedView style={styles.currentEventBox}>
+                <PrimaryText
+                  size={30}
+                  weight={FontWeight.SemiBold}
+                  label="Current Event"
+                />
+                <PrimaryText
+                  style={styles.currentEventTitle}
+                  size={25}
+                  weight={FontWeight.SemiBold}
+                  label={arbiterInfo.eventName}
+                />
+                <PrimaryText
+                  style={styles.currentEventDesc}
+                  size={20}
+                  weight={FontWeight.SemiBold}
+                  label={`Arbiter Name: ${arbiterInfo.userName}`}
+                />
+                <PrimaryText
+                  style={styles.currentEventDesc}
+                  size={20}
+                  weight={FontWeight.SemiBold}
+                  label={`Pin: ${arbiterInfo.pin}`}
+                />
+                <PrimaryButton
+                  style={styles.editEventButton}
+                  labelStyle={styles.editEventButtonLabel}
+                  onPress={() => setShowEditEvent(true)}
+                  label="Edit Event"
+                />
+              </RoundedView>
+            )}
+            {arbiterInfo === null && (
+              <RoundedView style={styles.currentEventBox}>
+                <PrimaryText
+                  size={30}
+                  weight={FontWeight.SemiBold}
+                  label="No Current Event"
+                />
+                <PrimaryText
+                  style={styles.currentEventTitle}
+                  size={20}
+                  weight={FontWeight.Medium}
+                  label="This ipad is not currently linked to an event, please set the event to continue"
```

Comment:
Super minor but it should be iPad (just so that it looks a bit more professional )